pyintegration/deephaven2/table.py

```diff
@@ -570,7 +569,10 @@ def sort_column(col, dir_):
         try:
             if order:
                 sort_columns = [sort_column(col, dir_) for col, dir_ in zip(order_by, order)]
-                return Table(j_table=_JTWD.sort(self.j_table, *sort_columns))
+                j_sc_list = _JArrayList()
+                for sc in sort_columns:
```

Comment:
Why don't we write a method "convert_list" or similar? Also, since we know the size of the pylist, we should size the arraylist.